redbot/cogs/mod/mutes.py

```diff
@@ -422,6 +423,8 @@ class MuteMixin(MixinMeta):
             await channel.set_permissions(user, overwrite=overwrites, reason=reason)
         except discord.Forbidden:
             return False, _(mute_unmute_issues["permissions_issue"])
+        except discord.NotFound:
+            return False, _(mute_unmute_issues["left_guild"])
```

Comment:
Could this also be if the channel was deleted as well? 🤣

Reply:
Added error handling for that issue.